utils/types.go

```diff
@@ -234,6 +237,9 @@ func GetKongClient(opt KongClientConfig) (*kong.Client, error) {
 	if err != nil {
 		return nil, fmt.Errorf("parsing headers: %w", err)
 	}
+	if opt.BearerToken != "" {
+		headers.Add("Authorization", "Bearer "+opt.BearerToken)
+	}
```

Comment:
Avoid konnect-specific additions in Kong client (unless a technical addition is required).
You can re-use opt.Headers for this purpose.

Reply:
Good point, thanks for the tip